typhon/cli.py

```diff
@@ -82,21 +82,30 @@ def create_suite(devices, cfg=None):
     else:
         logger.debug("Creating new happi Client from configuration")
         client = happi.Client.from_config(cfg=cfg)
-    logger.debug("Creating empty TyphonSuite ...")
-    suite = typhon.TyphonSuite()
-    logger.info("Loading Tools ...")
-    for name, tool in suite.default_tools.items():
-        suite.add_tool(name, tool())
     # Load and add each device
+    loaded_devs = list()
     for device in devices:
```

Comment:
Is it possible to enter this block with an empty devices list? If not, then this is a good change.

Reply:
So long as `devices` is an empty list/iterable and not `None`, it seems like this should be OK.

It will, however, return `None` instead of a suite as there will be no `loaded_devs`.

Reply:
`devices` is either entered by the user via `create_suite` or comes directly from `ArgumentParser` if executing via the command line. The latter is guaranteed to be iterable, the first is not, but I'm not sure I need to explicitly have an error message for this?

Also to @klauer point, the method that calls this watches for a `None` being returned. I didn't want to raise an exception here and then catch it somewhere else later. Whether this is the cleanest way to implement this is up for debate.

Reply:
There should definitely be a message about why nothing happened if it's possible for a user to get here. Otherwise it is fine as is.

Reply:
Point heard. This silently does nothing now which is the worst of both worlds. Now my question is if the user just calls the cli like this:

`typhon`

Do we?
* Count this an error, yell loudly you failed to enter any devices.
* Give them back an empty `TyphonSuite`. This option maybe more appealing someday if you can drag and drop new devices in from another application?

Reply:
Empty suite may be preferable?

Reply:
Done. Tests added as behavior is slightly complicated.